DominicGBauer/information-access
# feat: 147 Routing fix

# Pull Request

## Scope
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_workitems/edit/147

## Work Done
Routing to new URL - moving off localhost. Drawer closes on routing to new page. Style fixes for Drawer for mobile. Small logo icon has routing added. Removal of duplicate home.vue from resultcard folder.

## Steps to Test
```npm run start```
Click through drawer to ensure drawer closes after selecting FAQ and About page

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/136935004-bbf4562f-6031-4313-9ed8-7c78beef387a.png)
